test/system/220-healthcheck.bats

```diff
@@ -469,13 +469,14 @@ function _check_health_log {
 @test "podman healthcheck - stop container when healthcheck runs" {
     ctr="c-h-$(safename)"
     msg="hc-msg-$(random_string)"
+    hcStatus=$PODMAN_TMPDIR/hcStatus
 
     run_podman run -d --name $ctr             \
            --health-cmd "sleep 20; echo $msg" \
            $IMAGE /home/podman/pause
 
     timeout --foreground -v --kill=10 60 \
-        $PODMAN healthcheck run $ctr &
+        $PODMAN healthcheck run $ctr > $hcStatus &
```

Comment:
use `&>` to capture both stdout and stderr, in case of an error printed we will see it directly in the assert below